koocook_core/models/__init__.py

```diff
@@ -4,3 +4,6 @@
 from .review import AggregateRating, Comment, Rating
 from .tag import Tag, TagLabel
 from .user import Author, KoocookUser
+
+__all__ = ['MetaIngredient', 'RecipeIngredient', 'Post', 'Recipe', 'AggregateRating', 'Comment', 'Rating', 'Tag',
+           'TagLabel', 'Author', 'KoocookUser']
```

Comment:
I don't know whether it's editable outside the file or not. These in a tuple are fine too, I guess.

Reply:
Yes these can be edited outside the file. You'll need to import the file first though, which is when it is used, so not really a use editing them outside the file.